lib/v2/paradigm/writing.js

```diff
@@ -0,0 +1,49 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+
+const baseUrl = 'https://www.paradigm.xyz';
+
+module.exports = async (ctx) => {
+    const url = `${baseUrl}/writing`;
+
+    const response = await got(url);
+    const $ = cheerio.load(response.data);
+
+    const list = $('.Writing_writing__post___l5Qs')
+        .map((_, item) => ({
+            title: $(item).find('h2').text(),
+            link: `${baseUrl}${$(item).find('a').attr('href')}`,
+            author: $(item).find('span > a').text(),
+            pubDate: parseDate(
+                $(item)
+                    .find('p')
+                    .last()
+                    .text()
+                    .split(/\son\s/)
+                    .at(-1)
+            ),
+        }))
+        .get();
```

Comment:
Since this site is built atop nextjs, you can use the embedded json
```suggestion
    const nextData = JSON.parse($('#__NEXT_DATA__').text());
    const buildId = nextData.buildId;
    const list = nextData.props.pageProps.posts.map((item) => ({
        title: item.title,
        link: `${baseUrl}${item.slug}`,
        api: `${baseUrl}/_next/data/${buildId}${item.slug}.json`,
        author: item.authors.map((author) => author.name).join(', '),
        pubDate: parseDate(item.originalDate),
        category: item.tags,
    }));
```